manifests/istio-telemetry/kiali/templates/configmap.yaml

```diff
@@ -34,8 +34,14 @@ data:
         url_service_version: http://istio-pilot.{{ .Values.global.configNamespace }}:8080/version
       tracing:
         url: {{ .Values.kiali.dashboard.jaegerURL }}
+{{- if .Values.kiali.dashboard.inclusterjaegerURL }}
+        in_cluster_url: {{ .Values.dashboard.inclusterjaegerURL }}
```

Comment:
should this not be `.Values.kiali.dashboard.inclusterjaegerURL`?

Reply:
You are right. Cut&paste issue :(